src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java

```diff
@@ -114,6 +118,14 @@
             doc="VCFs for samples to include. May be specified either one at a time, or as one or more .args file containing multiple VCFs, one per line.", optional = false)
     private Set<File> vcfs = new LinkedHashSet<>(0);
 
+    /**
+     * The VCFs can be input as either one or more .args file(s) containing one VCF per line, or VCFs can be
```

Comment:
I think this comment block got copied and pasted from above

Reply:
Whoops!